static_test_driver/static_test_driver.ino

```diff
@@ -207,7 +214,6 @@ void loop() {
       for (int i = 0; i < NUM_HIST_VALS; i++)
```

Comment:
Mostly.  The stuff related to NUM_HIST_VALUES should go away, instead you should just call tare() after READ(flag_zero)